data/json/mapgen/parks_recreation/small_office.json

```diff
@@ -0,0 +1,124 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "small_office" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "________________________",
+        "________________________",
+        "||------||____||------||",
+        "|H.ssst.#|____|&..l...^|",
+        "-t......#|-DD-|..dd.#d.-",
+        "-H.tt...#-#ff#-...#..d.-",
+        "-^.tt....-#ff#-......l.-",
+        "-H.......D.ff.D..d#..d.-",
+        "-........-.aa.-^.d..#d.-",
+        "|FCC....&-.ff.-&.......|",
+        "||||+||+|||||||||||+||||",
+        "gg|..|&..........&|..|_g",
+        "gg|..|............|..h_g",
+        "gg|..|....|b|b|+|+|..|_g",
+        "||||+|SSSi|T|T|T|T|+||||",
+        "|&...||||||||||||||..#j|",
+        "|.............^|{....#v|",
+        "|.#.#.#.#.#....|{..d#.F|",
+        "|...........d#.|{.#d..#|",
+        "|.#.#.#.#.#.d..|{....dc|",
+        "|^............&|{..dccc|",
+        "|||--||||--||||||+||||||",
+        "ggggggggggkee_______gggg",
+        "ggggggggggg_________gggg"
+      ],
+      "terrain": {
+        "#": "t_linoleum_white",
+        "&": "t_linoleum_white",
+        "+": "t_door_c",
+        "-": "t_wall_glass",
+        ".": "t_linoleum_white",
+        "C": "t_linoleum_white",
+        "D": "t_door_glass_c",
+        "F": "t_linoleum_white",
+        "H": "t_linoleum_white",
+        "S": "t_linoleum_gray",
+        "T": "t_linoleum_gray",
+        "^": "t_linoleum_white",
+        "_": "t_sidewalk",
+        "a": "t_floor_red",
+        "b": "t_door_o",
+        "c": "t_linoleum_white",
+        "d": "t_linoleum_white",
+        "e": "t_sidewalk",
+        "f": "t_floor_red",
+        "g": "t_grass",
+        "h": "t_door_metal_c",
+        "i": "t_linoleum_gray",
+        "j": "t_linoleum_white",
+        "k": "t_grass",
+        "l": "t_linoleum_gray",
+        "s": "t_linoleum_white",
+        "t": "t_linoleum_white",
+        "v": "t_linoleum_white",
+        "{": "t_linoleum_white",
+        "|": "t_wall"
+      },
+      "furniture": {
+        "#": "f_chair",
+        "&": "f_trashcan",
+        "+": "f_null",
+        "-": "f_null",
+        ".": "f_null",
+        "C": "f_counter",
+        "D": "f_null",
+        "F": "f_fridge",
+        "H": "f_armchair",
+        "S": "f_sink",
+        "T": "f_toilet",
+        "^": "f_indoor_plant",
+        "_": "f_null",
+        "a": "f_desk",
+        "b": "f_null",
+        "c": "f_cupboard",
+        "d": "f_desk",
+        "e": "f_dumpster",
+        "f": "f_null",
+        "g": "f_null",
+        "h": "f_null",
+        "i": "f_trashcan",
+        "j": "f_dryer",
+        "k": "f_dumpster",
+        "l": "f_null",
+        "s": "f_sofa",
+        "t": "f_table",
+        "v": "f_vending_o",
+        "{": "f_locker",
+        "|": "f_null"
+      },
+      "place_toilets": [
+        { "x": 11, "y": 14 },
+        { "x": 13, "y": 14 },
+        { "x": 15, "y": 14 },
+        { "x": 17, "y": 14 }
+      ],
```

Comment:
Toilets would be much better mapped to a symbol. The only excuse not to do that is when a mapgen instance needs to have dry toilets mixed with non-dry ones. This isn't the case here.